Update readme with clearer issue reporting guidelines (#19964)
nvaccess/nvda: projectDocs/issues/readme.md

Issues can be [reported via GitHub](https://github.com/nvaccess/nvda/issues/new/choose).
Please read the [issue template guide](./githubIssueTemplateExplanationAndExamples.md) for help filling out the appropriate template.
For advanced reporters, consider reading our [issue triage guide](./triage.md) to ensure the issue can easily be worked on.

> If you don't have a clear idea of the issue, and you are unable to fill out the template, open a [GitHub discussion](https://github.com/nvaccess/nvda/discussions/new/choose) instead.

Reviewer: For that, the label would need to include “open a”, but it's better to leave the target as is, unless we want even more duplicates.
```suggestion
If you don’t have a clear idea of the issue, and you are unable to fill out the template, open a [discussion](https://github.com/nvaccess/nvda/discussions/new/choose) instead.
```

Author: this is better as-is

Do not report security concerns via GitHub issues, instead follow our [security policy](../../security.md).

Issues with materials handled by translators should be reported to the [NVDA Translators list](https://groups.io/g/nvda-translations).
These include:

* NVDA interface text that is incorrectly translated from English
* Contents of the User Guide and Changes documents that are incorrectly translated from English

> * Locale-specific gestures, symbols and character descriptions

Reviewer 2: ```suggestion
* Input gestures, punctuation/symbol pronunciations, and character descriptions in languages other than English
```

For advanced reporters, consider reading our [issue triage guide](./triage.md) to ensure the issue can easily be worked on.

> If you don't have a clear idea of the issue, and you are unable to fill out the template, open a [GitHub discussion instead](https://github.com/nvaccess/nvda/discussions).

Reviewer 3: Moved the word "instead" outside the anchor, and changed the link to point to the new discussion chooser.
```suggestion
If you don't have a clear idea of the issue, and you are unable to fill out the template, open a [GitHub discussion](https://github.com/nvaccess/nvda/discussions/new/choose) instead.
```

> Do not report security issues via GitHub issues, instead follow our [security policy](../../security.md).

Reviewer: “GitHub issues” are issues in GitHub. As already suggested:
```suggestion
For security issues, see our [security policy](../../security.md).
```

Author: As previously discussed, we want to explicitly state that these should not be reported via public GitHub issues.
Generally I try to avoid using "see" in our documentation, and rather "read" or "refer to".

> Do not report security issues via GitHub issues, instead follow our [security policy](../../security.md).

Reviewer 3: ```suggestion
Do not report security concerns via GitHub issues, instead follow our [security policy](../../security.md).
```

Issues with materials handled by translators should be reported to the [NVDA Translators list](https://groups.io/g/nvda-translations).

> This includes:

Reviewer 3: ```suggestion
These include:
```

* NVDA interface text that is incorrectly translated from English

> * User Guide and Changes documents/content that are incorrectly translated from English

Author: ```suggestion
* Contents of the User Guide and Changes documents that are incorrectly translated from English
```

> Issues with materials handled by translators should be reported to the [NVDA Translators list](https://groups.io/g/nvda-translations).
> This includes:
> 
> * NVDA interface text that is incorrectly translated from English
> * User Guide and Changes documents/content that are incorrectly translated from English
> * Locale-specific gestures, symbols and character descriptions

Reviewer: This looks just as confusing as the current situation. How does a contribuser know that a message (or other text) was incorrectly translated? In fact, how would they even know that it was translated from English? And what about what is not on that list?

As already explained, this should just be dropped, or perhaps rephrased as a suggestion.
```suggestion
```
```suggestion
If you know that you are reporting an issue specific to a translation, please also notify the [NVDA translations mailing list](https://groups.io/g/nvda-translations).
```

Author: These are just a list of examples, which helps with clarification as per the original issue.
I think most people can understand from this, that if their issue is a string that is not in English, they should report it on the list.

* NVDA interface text that is incorrectly translated from English

> * Contents of the User Guide and Changes documents that are incorrectly translated from English

Reviewer 2: ```suggestion
* NVDA interface text that is incorrect in languages other than English
* Contents of the User Guide and Changes documents that are incorrect in languages other than English
```
